components/ide/jetbrains/image/BUILD.js

```diff
@@ -0,0 +1,85 @@
+const ideConfigs = [
+    {
+        name: "intellij",
+        productCode: "IIU",
```

Comment:
should this be `IU`? [[1]](https://youtrack.jetbrains.com/issue/MP-2363/Can-you-provide-me-the-list-of-jetbrains-product-codes)